pkgs/by-name/lo/lovely-injector/package.nix

```
@@ -4,20 +4,20 @@
   lib,
 }:
 let
-  version = "0.6.0";
+  version = "0.7.1";
   lovelyInjector = fetchFromGitHub {
     owner = "vgskye";
     repo = "lovely-injector";
-    rev = "3224915f4d47b557c34b5012797cf92d4cc629af";
-    hash = "sha256-fzkuuu6pmvqeJa7qlX8jhtCLC4oYRLUm1hqHTRiYEX8=";
+    rev = "5dde1ff2647683d3c7fed748146720c30cc69826";
```

```suggestion
    tag = "v${version}";
```
Upstream tags releases